### DISCUSSION TITLE
Can you also proxy to embeddings endpoint?

### DISCUSSION THREAD

**Discussion by kengoodridge** [BerriAI/litellm#863] [Ideas]
Could you find the time to proxy to embeddings, supporting [openai api](https://platform.openai.com/docs/guides/embeddings/how-to-get-embeddings), specifically for [ollama](https://github.com/jmorganca/ollama/blob/main/docs/api.md#generate-embeddings), but I would guess you should do it for any of the llm providers that support it. I don't know how general embeddings support is. 

I think it would be particularly useful and clean for RAG.

Thanks! Find this package invaluable.

**Comment by ishaan-jaff** [comment]
Hi @kengoodridge thanks for the issue will work on this today 

here's our supported embedding models btw: https://docs.litellm.ai/docs/embedding/supported_embedding

**Comment by ghost** [comment]
This is now live - @kengoodridge 
https://github.com/BerriAI/litellm/blob/8291f239a47697e1b1de0c1bc1ec4d05d11a2d68/litellm/proxy/proxy_server.py#L607
